tests/Unit/GuzzleFileFetcherTest.php

```diff
@@ -85,31 +100,55 @@ public function errorProvider(): array
      *
      * @return void
      *
-     * @dataProvider errorProvider
+     * @dataProvider providerFetchFileError
      *
      * @covers ::fetchFile
      */
-    public function testError(int $statusCode, string $exceptionClass, ?int $exceptionCode = null, ?int $maxBytes = null): void
+    public function testfetchFileError(int $statusCode, string $exceptionClass, ?int $exceptionCode = null, ?int $maxBytes = null): void
```

Comment:
Nit, should be testFetchFileError (first 'f' is not capitalized)

Reply:
fixed